### PR TITLE
Allow Users to change customized ldb tools' header in help printing

### PR DESCRIPTION
**Pull request by siying**

### CODE REVIEW

**Comment by siying** [comment]
@IslamAbdelRahman @ajkr MyRocks is asking us to change the banner of the ldb tool. I'm going to allow users to plug in their own string and rename the name from LevelDB to RocksDB in default ldb.

File: include/rocksdb/ldb_tool.h [hunk 1]
```diff
@@ -26,6 +26,8 @@ struct LDBOptions {
   // Key formatter that converts a slice to a readable string.
   // Default: Slice::ToString()
   std::shared_ptr<SliceFormatter> key_formatter;
+
+  std::string print_help_header;
```
Comment (IslamAbdelRahman) [inline include/rocksdb/ldb_tool.h]: maybe we can just have a default message
```
std::string print_help_header = "A User-Customized RocksDB Tool";
```
and get rid of the if/else

**Comment by ajkr** [comment]
i can't think of why anyone would need configurable help header

**Comment by siying** [comment]
@ajkr talking to you offline for this question.

**Pull request closed without merging**